pymc3/distributions/continuous.py

```diff
@@ -765,7 +728,7 @@ def logp(
         else:
             norm = 0.0
 
-        logp = Normal.logp(value, mu=mu, sigma=sigma) - norm
+        logp = _logprob(normal, [value], *(0, 0, 0, mu, sigma)) - norm
```

Comment:
```suggestion
        logp = _logprob(normal, (value,), None, None, None, mu, sigma) - norm
```
Seems less error prone